docs/user_guide.md

```diff
@@ -1519,6 +1521,11 @@ optional arguments:
                         then "run_2*:run_3_d_name" selects the last three runs.
                         Use 'CodeChecker cmd runs' to get the available runs.
   -a, --all             Show breakdown for all analysis runs.
+  --disable-unique      Narrows the report to list all bugs. The same bug may
```

Comment:
Is narrows really the right term here? As the result of passing this flag I expect to get more reports, not less.